frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx

```diff
@@ -207,6 +207,7 @@ export interface LineGraphProps {
     showPersonsModal?: boolean
     tooltip?: TooltipConfig
     inCardView?: boolean
+    inSurveyView?: boolean
```

Comment:
These are nicer as specific customisation options, like `hideGrid` and `hideAxis`

Reply:
clearer to understand, and then not limited to just surveys for whoever uses this component in the future

Reply:
I agree and added `hideXAxis` and `hideYAxis` props.

I still had to leave `inSurveyView` in two places - the logic in the component is convoluted and could use a refactor - but not sure if it's worth it at this point since Cory mentioned he's planning to rethink our visualization components.